package.json

```diff
@@ -14,17 +14,12 @@
 		"packages/*"
 	],
 	"devDependencies": {
-		"@wordpress/env": "^10.28.0",
-		"@wordpress/scripts": "^30.21.0",
-		"@wordpress/prettier-config": "^4.28.0",
-		"@testing-library/jest-dom": "^6.1.5",
-		"@testing-library/react": "^14.1.2",
-		"@types/jest": "^30.0.0",
-		"@wordpress/eslint-plugin": "^22.16.0",
-		"@wordpress/jest-console": "^8.30.0",
-		"eslint": "^8.57.0",
-		"eslint-plugin-import": "^2.32.0",
-		"eslint-plugin-jest": "^27.9.0",
+		"@types/jest": "^29.5.14",
+		"@wordpress/env": "^10.31.0",
+		"@wordpress/eslint-plugin": "^22.17.0",
+		"@wordpress/jest-console": "^8.31.0",
```

Comment:
We most likely don't have to include `@wordpress/jest-console` as it isn't referenced in the code.